draft-enghardt-panrg-path-properties.md

```diff
@@ -46,8 +48,25 @@ This information is expressed as properties of paths between two endpoints.
+In this document, we specify a set of path properties which might be useful in the following use cases: Traffic policies, network monitoring, and path selection.
+
+- Traffic policies: Network operators or end users can define traffic policies leveraging path awareness.
+Such policies can allow or disallow sending traffic over specific networks or nodes, select an appropriate protocol depending on the capabilities of the on-path devices, or adjust protocol parameters to an existing path.
+An example of a traffic policy is a video streaming application choosing an (initial) video quality based on the achievable data rate, or the monetary cost of the link using a volume-based or flat-rate cost model.
+Another example is an enterprise network where all traffic has to go through a firewall, in which case the endpoint needs to be aware of on-path firewalls.
+
+- Network monitoring: Network operators can use path properties, (e.g., measured by on-path devices), to check whether recent end-user traffic satisfies specific Quality of Service (QoS) requirements, and identify potential problems with their network early on, before the end-user complains.
+
+- Path selection: In some cases, entities can choose to use a certain path (or subset of paths) from a set of paths to achieve a specific goal.
+The possible benefits of a well chosen path varies based on the goal, and thus the only guarantee that a path aware network should give is to not perform worse than the default case without path selection.
+Depending on the goal, paths with different properties are preferred, e.g., retrieving a webpage as quickly as possible requires low latency paths, retrieving a large file in a peer-to-peer network requires paths with high achievable data rate.
+A network (e.g., an AS) can adjust its path selection for internal or external routing based on the path properties.
+In BGP, the Multi Exit Discriminator (MED) attribute decides which path to choose if other attributes are equal; in a path aware network, instead of using this single MED value, other properties such as maximum or available/expected data rate could additionally be used to improve load balancing.
+An endpoint might be able to select between a set of paths, either if there are several paths to the same destination (e.g., if the endpoint is a mobile device with two wireless interfaces, both providing a path), or if there are several destinations, and thus several paths, providing the same service (e.g., Application-Layer Traffic Optimization (ALTO) {{RFC5693}}, an application layer peer-to-peer protocol allowing endpoints a better-than-random peer selection).
+Care needs to be taken when selecting paths based on path properties, as some path properties that depend on the current state of the network can become outdated and not related to path properties of packets sent now.
```

Comment:
This still reads a bit confusing to me, because we have "the current state of the network" and "packets sent now", which sounds like we are talking about the same point in time.
Suggestion: "[…] as path properties that were recently measured may have become outdated and, thus, useless to predict the path properties of packets sent now."

Reply:
Changed sentence to:
[...], as path properties that were previously measured may have become outdated and thus useless to predict the path properties of packets sent now.

Reply:
Yes, that works! My professor is very strict about always putting commas around the "thus", but I guess we don't have to be so strict here ;)